### ISSUE TITLE
ZeroDivisionError on uploading empty dataframe

### ISSUE THREAD

**Issue by knowsuchagency** [pydata/pandas-gbq#237]
Attempting to upload an empty dataframe seems to raise to a `ZeroDivisionError` in version `0.8.0`

```
[2018-12-01 09:40:32,542] {base_task_runner.py:95} INFO - Subtask:   File "/usr/local/lib/python2.7/site-packages/pandas_gbq/gbq.py", line 509, in load_data
[2018-12-01 09:40:32,542] {base_task_runner.py:95} INFO - Subtask:     ((total_rows - remaining_rows) * 100) / total_rows
[2018-12-01 09:40:32,542] {base_task_runner.py:95} INFO - Subtask: ZeroDivisionError: integer division or modulo by zero
```

**Comment by max-sixty** [comment]
Thanks. Would def take a PR for this.

**Comment by tswast** [comment]
Closed by https://github.com/pydata/pandas-gbq/pull/252.

Will be fixed in the 0.10.0 release.

**Comment by tswast** [comment]
Not entirely fixed yet, as I noted in https://github.com/pydata/pandas-gbq/pull/252/files#r263975138 the load job can fail when a DataFrame contains no rows and no columns.
